specs/waku/waku-rpc-api.md

```diff
@@ -0,0 +1,410 @@
+---
+title: Waku RPC API
+version: 1.0.0
+status: Draft
```

Comment:
Is this a Draft? Vs stable

If draft version seems off

Also should other specs be updated to point to this?

Reply:
^

Reply:
re other specs

Reply:
Where should we point to this? In the waku spec?